lib/ecto/multi.ex

```diff
@@ -602,6 +602,18 @@ defmodule Ecto.Multi do
   defp format_operation(other),
     do: other
 
+  @doc """
+  Adds a value to the changes so far under the given name.
+
+  ## Example
+
+      Ecto.Multi.put(multi, :password, "sghbj15")
```

Comment:
```suggestion
      Ecto.Multi.new()
      |> Ecto.Multi.put(:params, params)
      |> Ecto.Multi.run()
      |> Ecto.Multi.insert(:user, fn changes -> &User.changeset(changes.params) end)
      |> Ecto.Multi.insert(:person, fn changes -> &Person.changeset(changes.user, changes.params) end)
      |> MyApp.Repo.transaction()
```

I think this could show how it would be useful